x/wasm/keeper/gas_register.go

```diff
@@ -26,7 +26,8 @@ const (
 	DefaultEventAttributeDataCost uint64 = 1
 	// DefaultContractMessageDataCost is how much SDK gas is charged *per byte* of the message that goes to the contract
 	// This is used with len(msg)
-	DefaultContractMessageDataCost uint64 = 1
+	// Note: also used for error fields on reply, and data on reply
```

Comment:
Maybe we should add that the contract charges gas when deserializing those messages.